### DISCUSSION TITLE
Import a Toonsquid symbol to another project

### DISCUSSION THREAD

**Discussion by ivana67dj** [keiwando/toonsquid#64] [Q&A]
Hello! I was wondering if there is a way to make a symbol in one project and then use it in another project

**Comment by MojoMan49** [comment]
Thanks and I wish the developer would enable this. One way would be to make the “symbol library” accessible to any and all projects in the App.

This for me is far more urgent than tweaking painting tools (which while appreciated needs to be repiroitized)
